src/components/ebay-checkbox/index.js

```diff
@@ -1,11 +1,16 @@
 
 module.exports = require('marko-widgets').defineComponent({
     template: require('./template.marko'),
-    handleChange(originalEvent) {
-        this.emit('checkbox-change', {
+    handleChange: forwardEvent('change'),
+    handleFocus: forwardEvent('focus')
+});
+
+function forwardEvent(eventName) {
+    return function(originalEvent, el) {
+        this.emit(`checkbox-${eventName}`, {
             originalEvent,
-            value: originalEvent.target.value,
-            checked: originalEvent.target.checked
+            value: (el || this.el.querySelector('input')).value,
```

Comment:
Since this is on the input itself, does it make sense to change the value selector?
Also did you test that this is giving the right value?

Reply:
Yes, and I have updated the focus event test to validate the value.